### PR TITLE
Fix expand eic widget

### PR DESCRIPTION
**Pull request by sahil21**

### CODE REVIEW

File: mzroll/mainwindow.cpp [hunk 1]
```diff
@@ -2885,201 +2885,54 @@ QWidget* MainWindow::eicWidgetController() {
...
-	}
-
-	//TODO: Sahil-Kiran, Added while merging mainwindow
-    QToolButton *btnShowSplines = new QToolButton(toolBar);
-    btnShowSplines->setIcon(QIcon(rsrcPath + "/splines.png"));
-    btnShowSplines->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
-    btnShowSplines->setToolTip(tr("Show Splines"));
-    btnShowSplines->setCheckable(true);
-    btnShowSplines->setChecked(false);
-	btnShowSplines->setVisible(false);
```
Comment (rkdahmiwal) [inline mzroll/mainwindow.cpp]: This button "btnShowSplines" was not in develop branch or might be the case it was not visible. But right now in this branch this button is added(or visible) in the end on toolbar of eicWidget.

File: mzroll/mainwindow.cpp [hunk 2]
```diff
@@ -3093,6 +2946,245 @@ QWidget* MainWindow::eicWidgetController() {
...
+	}
+	else if (btnName == "btnShowSplines") {
+
+		QToolButton *btnShowSplines = new QToolButton(parent);
+		btnShowSplines->setIcon(QIcon(rsrcPath + "/splines.png"));
+		btnShowSplines->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
+		btnShowSplines->setToolTip(tr("Show Splines"));
+		btnShowSplines->setCheckable(true);
+		btnShowSplines->setChecked(false);
+		btnShowSplines->setVisible(false);
```
Comment (rkdahmiwal) [inline mzroll/mainwindow.cpp]: This button "btnShowSplines" was not in develop branch or might be the case it was not visible. But right now in this branch this button is added(or visible) in the end on toolbar of eicWidget.

File: mzroll/mainwindow.cpp [hunk 3]
```diff
@@ -3093,6 +2946,245 @@ QWidget* MainWindow::eicWidgetController() {
...
+		QToolButton *btnShowTic = new QToolButton(parent);
+		btnShowTic->setCheckable(true);
+		btnShowTic->setChecked(false);
+		btnShowTic->setIcon(QIcon(rsrcPath + "/tic.png"));
+		btnShowTic->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
+		btnShowTic->setToolTip(tr("Show TICs"));
+		connect(btnShowTic, SIGNAL(toggled(bool)), mw->getEicWidget(), SLOT(replot()));
+		connect(btnShowTic, SIGNAL(toggled(bool)), mw->getEicWidget(),
+				SLOT(showTicLine(bool)));
+
```
Comment (rkdahmiwal) [inline mzroll/mainwindow.cpp]: The behaviour of this button is changed as compared with the same button of develop branch. Have a look.

File: mzroll/mainwindow.cpp (hunk 2 above)
Comment (sahil21) [inline mzroll/mainwindow.cpp]: @rkdahmiwal Resolved.

File: mzroll/mainwindow.cpp (hunk 3 above)
Comment (sahil21) [inline mzroll/mainwindow.cpp]: @rkdahmiwal Resolved. Problem was with connect statement order

**Pull request merged**